.gitignore

```diff
@@ -17,6 +17,7 @@ docs/notebooks/.ipynb_checkpoints/
 docs/_autosummary
 .idea
 .vscode
+.envrc
```

Comment:
I suspect this was accidental but I don't mind including it.

Reply:
Sorry, I forgot to remove that. But indeed I'd be **very** happy if this stayed around..